### ISSUE TITLE
`solara.display` doesn't work in flask

### ISSUE THREAD

**Issue by iisakkirotko** [widgetti/solara#500]
If `solara.display` is used when running solara with flask we only get a javascript error:

`Javascript Error: Cannot read properties of undefined (reading 'sanitize')`

**Comment by egormkn** [comment]
I'm getting the same error when running Solara in production mode. In development mode everything is ok.
Output widget from ipywidgets also doesn't work in production mode.
Here is a stack trace:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'sanitize')
    at Object._ (webpack://widgetti/solara-widget-manager8/node_modules/@jupyterlab/rendermime/lib/renderers.js:393:31)
    at p.render (webpack://widgetti/solara-widget-manager8/node_modules/@jupyterlab/rendermime/lib/widgets.js:317:16)
    at p.renderModel (webpack://widgetti/solara-widget-manager8/node_modules/@jupyterlab/rendermime/lib/widgets.js:49:20)
    at p._setOutput (webpack://widgetti/solara-widget-manager8/node_modules/@jupyterlab/outputarea/lib/widget.js:414:27)
    at p.onModelChanged (webpack://widgetti/solara-widget-manager8/node_modules/@jupyterlab/outputarea/lib/widget.js:269:22)
    at u (webpack://widgetti/solara-widget-manager8/node_modules/@lumino/signaling/dist/index.es6.js:510:18)
    at e.emit (webpack://widgetti/solara-widget-manager8/node_modules/@lumino/signaling/dist/index.es6.js:465:17)
    at e.emit (webpack://widgetti/solara-widget-manager8/node_modules/@lumino/signaling/dist/index.es6.js:121:21)
    at u._onListChanged (webpack://widgetti/solara-widget-manager8/node_modules/@jupyterlab/outputarea/lib/model.js:231:23)
    at u (webpack://widgetti/solara-widget-manager8/node_modules/@lumino/signaling/dist/index.es6.js:510:18)
```

**Comment by iisakkirotko** [comment]
Thanks a lot @egormkn, your observation that this happens in production mode was very useful in finding the culprit!

**Comment by iisakkirotko** [comment]
Should be fixed in `solara==1.32.1`
